proto/nft/v1/tx.proto

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2016-2021 Shanghai Bianjie AI Technology Inc. (licensed under the Apache License, Version 2.0)
+// Modifications Copyright (c) 2021, CRO Protocol Labs ("Crypto.org") (licensed under the Apache License, Version 2.0)
+syntax = "proto3";
+package chainmain.nft.v1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/crypto-org-chain/chain-main/x/nft/types";
+option (gogoproto.goproto_getters_all) = false;
+
+// Msg defines the htlc Msg service.
+service Msg {
+    // IssueDenom defines a method for issue a denom.
+    rpc IssueDenom(MsgIssueDenom) returns (MsgIssueDenomResponse);
+
```

Comment:
we can probably keep the names, but just may need some comments to note that "Denom == Collection" in here?